src/homework5/homework5.sql

```diff
@@ -0,0 +1,113 @@
+USE baseball;
+
+DROP TABLE IF EXISTS away_team_batter_stats;
+
+CREATE TABLE away_team_batter_stats AS
+SELECT
+    game_id,
+    team_id,
+    atBat AS AB,
+    Hit AS H,
+    Single AS singles, -- using plural because 'Double' is a key word
+    team_batting_counts.Double AS doubles, -- have to use team_batting_counts because 'Double' is a key word
+    Triple AS triples,
+    Home_Run AS HR,
+    Hit_By_Pitch AS HBP,
+    Sac_Fly AS SF,
+    Walk AS BB
+FROM team_batting_counts
+WHERE homeTeam = 0;
+
+CREATE INDEX away_team_batter_stats_game_id_idx ON away_team_batter_stats (game_id);
```

Comment:
really good that you created the index so it reduces the running time for the rest of the script. good job!